### ISSUE TITLE
Having fun with laser pointers.

### ISSUE THREAD

**Issue by MomoiroShinobinomono** [CleverRaven/Cataclysm-DDA#49854]
# Is your feature request related to a problem? Please describe.
It would be nice to have laser pointers in game.

# Describe the solution you'd like
Add two types of laser pointers.
1. Red laser pointers, which can be played with for a couple of minutes to give minor morale boost. I suppose the most logical place to find them would be zombie children' belongings.

2. Green/blue laser pointers which can be used to start a fire or temporarily blind an enemy.
Chance to blind depends on Marksmanship skill and a Perception check.
They should only be compatible with the smallest batterie types.
Only enemies with SEES flag can be blinded.
It wont work on enemies with an ELECTRONIC flag. Or will, game balance is not my area of expertise sorry.
You cannot use it while either you or your target are in smoke.
It can be used underwater but your chances to hit will be, I don't know, 80% of the original chance? You know, differing water salinity, water temperature layers, all that stuff can slightly refract a laser beam in real life.
You can aim and shoot them at mirrors which will roll a standard hit check and on success cause temporary blindness effect to be applied on a player character if said character doesn't have welding goggles/mask on.
When they're used at dark places the tile our character stands on will be illuminated for a duration it takes to perform a blinding attempt. As when they're used at night in real life, it is impossible not to notice the direction a laser beam is coming from.
I suppose it's okay if green laser pointer usage doesn't raise any of player character's skills.
In USA they can be bought at Walmart so the most likely location to find them should be an electronics store I suppose.

**Basically a 0 damage gun that is dangerous to use at dark places, has high firing range, an ability to start a fire, and an extremely limited ammo.**

_There could be a real world reference in a form of an addition of one green laser pointer to an inventory of a pulped human body that will spawn with it near airplane crash overworld map tile. Just to remind our fellow players that shooting a laser beam at a flying aircraft is not only illegal but also extremely stupid and dangerous._

**Comment by actual-nh** [comment]
> Red laser pointers, which can be played with for a couple of minutes to give minor morale boost. I suppose the most logical place to find them would be zombie children' belongings.

Enhanced morale bonus from playing with a pet (or at least some cats...)?

**Comment by MomoiroShinobinomono** [comment]
> Enhanced morale bonus from playing with a pet (or at least some cats...)?

Yeah, that sounds fun!

_Though, as a side note, in real life prolonged laser pointer play might heavily f'ck up pet's normal behavioral patterns, be it in dogs or in cats._

**Comment by MomoiroShinobinomono** [comment]
Oh wait the title makes it look like a "Failed Rationalization", sorry.

**Comment by capSAR273** [comment]
Maybe batteries instead of ammo? The more powerful ones could use rechargeable ones and the weak ones could use disposable. Might need to pair the blue/green lasers with a lens to start fires, unless they were extremely powerful (and would probably blind the player while using them).

Wearing laser goggles could be required to offset the benefit of being able to quickly blind a group of enemies. LED Diode lasers powerful enough to start fires on their own will definitely blind someone even if they didn't point it at their own eyes.

**Comment by MomoiroShinobinomono** [comment]
I don't want them to become a trick which could get you out of any situation so I don't think that giving avatars an ability to quickly blind groups of enemies with it is a good idea.

**Comment by Night-Pryanik** [comment]
Closing as stale, since stalebot can't do this by itself.
